frontend/src/metabase/query_builder/components/filters/pickers/DatePicker/DatePicker.unit.spec.tsx

```diff
@@ -356,5 +356,30 @@ describe("DatePicker", () => {
         });
       });
     });
+
+    describe("Exclude", () => {
+      it("should correctly update exclude filter when value is 0 even though it is falsy", async () => {
+        const onChangeMock = jest.fn();
+
+        render(
+          <DatePickerStateWrapper filter={filter} onChange={onChangeMock} />,
+        );
+
+        userEvent.click(screen.getByText("Exclude..."));
+        userEvent.click(screen.getByText("Hours of the day..."));
+
+        expect(screen.getByTestId("exclude-0")).not.toBeChecked();
```

Comment:
Shouldn't the option be checked initially?

```suggestion
        expect(screen.getByTestId("exclude-0")).toBeChecked();
```

Reply:
Good catch, thanks! I updated the test